### DISCUSSION TITLE
How to use SquirrelJME in RetroArch

### DISCUSSION THREAD

**Discussion by Gabrielcnnunes** [SquirrelJME/SquirrelJME#218] [General Questions]
How to use this emulator/core in RetroArch? Which bios? Which extension files does this reads? Please is there a tutorial on how to use this? I could never use it.

**Comment by XerTheSquirrel** [comment]
Hi! This is still a WIP!

Regarding the extensions it will be: `jar|sqc|jam|jad|kjx|adf|sec|sto`.
